beacon_node/network/src/sync/manager.rs

```diff
@@ -965,15 +966,31 @@ impl<T: BeaconChainTypes> SyncManager<T> {
     fn on_data_columns_by_root_response(
         &mut self,
         req_id: DataColumnsByRootRequestId,
-        _requester: SingleLookupReqId,
+        requester: SingleLookupReqId,
         peer_id: PeerId,
         rpc_event: RpcEvent<Arc<DataColumnSidecar<T::EthSpec>>>,
     ) {
-        if let Some(_resp) = self
+        if let Some(resp) = self
             .network
             .on_data_columns_by_root_response(req_id, peer_id, rpc_event)
         {
-            // TODO(das): pass data_columns_by_root result to consumer
+            if let Some(custody_columns) = self
+                .network
+                .on_custody_by_root_response(requester, req_id, peer_id, resp)
+            {
+                // TODO(das): get proper timestamp
+                let seen_timestamp = timestamp_now();
+                self.block_lookups
+                    .on_download_response::<CustodyRequestState<T::EthSpec>>(
+                        requester,
+                        // TODO(das): this is not the correct peer to attribute to a response. It's
+                        // just the last peer of multiple potential requests. We should switch to a
+                        // concept of "peer group" to attribute fault to the correct peer.
+                        peer_id,
```

Comment:
We already have this on `das` branch, is there a reason why you don't want it included in this PR?